### PR TITLE
Downloader updates

### PR DESCRIPTION
**Pull request by StefanoRaggi**
- All: Changed Symbol.ToString() to Symbol.Value
- YahooDownloader: fixed wrong month in url query
- YahooDownloader: added StartDate/EndDate as command line args
- YahooDownloader: added command line option to download multiple symbols
- GoogleDownloader: added StartDate/EndDate as command line args
- GoogleDownloader: added command line option to download multiple symbols

### CODE REVIEW

File: ToolBox/GoogleDownloader/GoogleDataDownloader.cs [hunk 1]
```diff
@@ -51,12 +51,15 @@ public IEnumerable<BaseData> Get(Symbol symbol, SecurityType type, Resolution re
             if (type != SecurityType.Equity)
                 throw new NotSupportedException("SecurityType not available: " + type);
 
+            if (endUtc < startUtc)
+                throw new ArgumentException("The end date must be greater or equal than the start date.");
+
             var numberOfDays = (int)(endUtc - startUtc).TotalDays;
             var resolutionSeconds = (int)resolution.ToTimeSpan().TotalSeconds;
-            var startUnixTime = ToUnixTime(startUtc);
+            var startUnixTime = ToUnixTime(endUtc);
```
Comment (mchandschuh) [inline ToolBox/GoogleDownloader/GoogleDataDownloader.cs]: Did we intend to assign the end to start?

**Comment by mchandschuh** [comment]
Great changes @StefanoRaggi - I had just one minor comment in GoogleDataDownloader.cs

File: ToolBox/GoogleDownloader/GoogleDataDownloader.cs (hunk 1 above)
Comment (StefanoRaggi) [inline ToolBox/GoogleDownloader/GoogleDataDownloader.cs]: Yes, it actually is an end date. I'll rename it to endUnixTime.

**Comment by mchandschuh** [comment]
Merged in https://github.com/QuantConnect/Lean/commit/69ab5193083899cb94d95265cc43092f75a84b70

**Pull request merged**